src/library/scala/collection/immutable/Vector.scala

```diff
@@ -639,29 +575,22 @@ override def companion: GenericCompanion[Vector] = Vector
     val blockIndex = (cutIndex - 1) & ~31
     val xor = startIndex ^ (cutIndex - 1)
     val d = requiredDepth(xor)
-    val shift = (startIndex & ~((1 << (5*d))-1))
-
-/*
-    println("cut back at " + startIndex + ".." + cutIndex + " (xor: "+xor+" d: " + d +")")
-    if (cutIndex == blockIndex + 32)
-      println("OUCH!!!")
-*/
-    val s = new Vector(startIndex-shift, cutIndex-shift, blockIndex-shift)
+    val shift = startIndex & ~((1 << (5 * d)) - 1)
+
+    val s = new Vector(startIndex - shift, cutIndex - shift, blockIndex - shift)
     s.initFrom(this)
     s.dirty = dirty
     s.gotoPosWritable(focus, blockIndex, focus ^ blockIndex)
     s.preClean(d)
-    s.cleanRightEdge(cutIndex-shift)
+    s.cleanRightEdge(cutIndex - shift)
     s
   }
-
 }
 
-
 class VectorIterator[+A](_startIndex: Int, endIndex: Int)
-extends AbstractIterator[A]
-   with Iterator[A]
-   with VectorPointer[A @uncheckedVariance] {
+  extends AbstractIterator[A]
+    with Iterator[A]
+    with VectorPointer[A @uncheckedVariance] {
```

Comment:
This style makes it hard to tell at a glance where the class signature ends and where the body begins.  The style is all over the place in practice, but I recommend left-justifying everything; vertical whitespace should always be used to separate classes so there is no need to indent them to indicate that they all belong to `VectorIterator`.  Failing that, the previous version was preferable because at least all the extended classes/traits are lined up.